### ISSUE TITLE
Can you provide more detailed information about the training set and validation set

### ISSUE THREAD

**Issue by zlr-zmm** [Zhou-Yetong/Opt_PredLLPS#1]
Thank you for your outstanding work. During my reading, I was very puzzled about the use of the dataset. Can the author provide a more detailed explanation of the pairing between the training set and the test set. I really want to know which ones are the training set and which ones are the testing set, and which models trained on these training sets are used on that testing set. If you could answer my questions, I would be extremely grateful.
